### ISSUE TITLE
How to enable RTX

### ISSUE THREAD

**Issue by ozdeadmeat** [itzg/docker-minecraft-bedrock-server#308]
Is there a way to enable RTX with this server?

**Comment by itzg** [comment]
I'm pretty sure that's a client side resource pack (to get the RTX aware textures) along with ensuring the client side parameter is set.

**Comment by Grekkq** [comment]
Yes it is only client side thing, if RTX resource packs are installed and enabled it should work.
@ozdeadmeat I have just tested it with `Vanilla RTX` pack (following the guide from [here](https://mcpedl.com/truly-vanilla-rtx/)) and it works right away, so if it works for you when you launch local game it should work on any server provided they do not enforce certain resource pack.
